tsdb/head_bench_test.go

```diff
@@ -79,6 +86,71 @@ func BenchmarkHeadStripeSeriesCreate_PreCreationFailure(b *testing.B) {
 	}
 }
 
+func BenchmarkHead_WalCommitWithVaryingSeriesSamplesExemplarsAndHistograms(b *testing.B) {
+	minute := func(m int) int64 { return int64(m) * time.Minute.Milliseconds() }
+	seriesCounts := []int{100, 1000, 10000}
+	series := genSeries(10000, 10, 0, 0)
+	histograms := genHistogramSeries(10000, 10, minute(0), minute(119), minute(1), true)
```

Comment:
```suggestion
	seriesCounts := []int{100, 1000, 10000}
	series := genSeries(10000, 10, 0, 0)
	histograms := genHistogramSeries(10000, 10, 0, 119, 1, true)
```

Suggested something mostly because I couldn't understand the goal of that code 🤔